Pages/avatar.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="../style.css">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+		integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+	<title>FreshStyles-UI</title>
+</head>
+
+<body>
+
+
+	<!-- horizontal Nav-bar -->
+	<div class="navHead cont-shadow">
+		<a href="/">
+			<div class="logo title-xxl mg-2-lt">FreshStyles-UI</div>
+		</a>
+		<span class="nav-text">Feel The Freshness...</span>
+		<div class="connect">
+			<i class="fab fa-github fa-lg">
+				<a href="/"></a>
+			</i>
+			<i class="fab fa-linkedin fa-lg">
+				<a href="/"></a>
+			</i>
+			<i class="fab fa-twitter-square fa-lg">
+				<a href="/"></a>
+			</i>
+		</div>
+
+	</div>
+
+
+	<!-- nav verticle bar -->
+	<nav class="nav mg-2-lr mg-1-top cont-shadow">
+		<h1 class="title-lg pd-point8-lt nav-sub-head hover-underline-animation">
+			Components
+		</h1>
+		<a href="./avatar.html"><button class="btn secondary-text-btn-lg-nav">Avatar</button>
+		</a>
+		<a href="./alert.html"><button class="btn secondary-text-btn-lg-nav">Alert</button>
+		</a>
+		<a href="./badge.html"><button class="btn secondary-text-btn-lg-nav">Badge</button>
+		</a>
+		<a href="./button.html"><button class="btn secondary-text-btn-lg-nav">Button</button>
+		</a>
+		<a href="./cards.html"><button class="btn secondary-text-btn-lg-nav">Cards</button>
+		</a>
+		<a href="./image.html"><button class="btn secondary-text-btn-lg-nav">Image</button>
+		</a>
+		<a href="./input.html"><button class="btn secondary-text-btn-lg-nav">Input</button>
+		</a>
+	</nav>
+
+
+	<!---------------------------------------------- Avatar ----------------------------------------------------------->
+	<section id="avatar" class="pd-1-lr pd-1-bot">
+		<h1 class="mg-point8-tb ">
+			<div class="sub-head">
+				Avatar
+			</div>
+		</h1>
+		<p>
+			An avatar is a graphical representation of a user or the user's character or persona. Aspernatur esse
+			libero, enim iure numquam consequatur tempore officiis nesciunt nobis possimus illo fugit repellendus
+			ducimus laboriosam maxime beatae nostrum placeat! Numquam!
```

Comment:
what it is for ? can't see it on UI

Reply:
- Will Repair it ...Thanks for pointing it✅